.github/workflows/transcription.yml

```diff
@@ -29,6 +32,7 @@ jobs:
       - name: Run transcription tests
         run: |
           npm run mocha -- --exit --bail "packages/tests/src/transcription/**/*.spec.ts"
+          yarn run benchmark-transcription
```

Comment:
I think we can remove the benchmark from the CI

Reply:
I might as well remove the `.github/workflows/transcription.yml` file altogether.  What do you think ?

Reply:
As for the Python dependencies I'll add `"preinstall": "pip install -r requirements.txt"` to the transcription package.

Reply:
> I might as well remove the .github/workflows/transcription.yml file altogether. What do you think ?

Yes please!

Reply:
Not sure about the "test" workflow :
- which test suite will run the transcription package test ?
- shall I execute the preinstall script manually somewhere ?

Reply:
You can create another test case here: https://github.com/Chocobozzz/PeerTube/blob/develop/scripts/ci.sh where you install your dependencies 

Then just add the it to the matrix: https://github.com/Chocobozzz/PeerTube/blob/develop/.github/workflows/test.yml#L42